google-cloud-logging/test/helper.rb

```diff
@@ -73,14 +73,15 @@ def random_entry_hash
       "severity"  => :DEFAULT,
       "insert_id" => "abc123",
       "labels" => {
-        "env" => "production",
-        "foo" => "bar"
+        "env"  => "production",
+        "foo"  => "bar"
```

Comment:
Nit: Is this necessary?

Reply:
No. None of the hash spacing is neccessary. But I think it looks better when it is consistent across lines.

Reply:
Usually I see the spacing being different at different "levels" of the hash. But, meh, LGTM or whatever. :)